internal/entity/album.go

```diff
@@ -125,6 +125,17 @@ func AddPhotoToUserAlbums(photoUid string, albums []string, userUid string) (err
 			if err = entry.Save(); err != nil {
 				log.Errorf("album: %s (add photo %s to albums)", err.Error(), photoUid)
 			}
+
+			// save the album with updated value for updated_at
+			m := Album{}
```

Comment:
See the code I've just pushed for how to update just the `updated_at` column without loading the record first and then indirectly updating it through the hooks that `Save()` calls.